sdk/python/kfp/v2/compiler_tests/test_data/simple_pipeline_without_importer.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pathlib
+
+import kfp
+from kfp.v2 import dsl
+import kfp.v2.compiler as compiler
+
+component_op_1 = kfp.components.load_component_from_text("""
+name: Write to GCS
+inputs:
+- {name: text, type: String, description: 'Content to be written to GCS'}
+outputs:
+- {name: output_gcs_path, type: GCSPath, description: 'GCS file path'}
+implementation:
+  container:
+    image: google/cloud-sdk:slim
+    command:
+    - sh
+    - -c
+    - |
+      set -e -x
+      echo "$0" | gsutil cp - "$1"
+    - {inputValue: text}
+    - {outputPath: output_gcs_path}
+""")
+
```

Comment:
nit: Given the difference in the behavior b/w v1 and v2, it'll be great if we can document here saying that the `output_gcs_path` will be modeled as an artifact and passed by reference between pipeline tasks.

Reply:
Given this currently live under `test_data` instead of `sample`, it's probably not the best place for documentation. 

Besides, given our open discussion on whether GCSPath alone is a good type, I would lean towards delaying the documentation.

Reply:
SGTM